lib/bbbevents/recording.rb

```diff
@@ -58,6 +59,7 @@ def initialize(events_xml)
       @attendees.values.each do |att|
         att.leaves << @finish if att.joins.length > att.leaves.length
         att.duration = total_duration(att)
+        att.duration_human = Time.at(att.duration).utc.strftime("%H:%M:%S")
```

Comment:
Using `Time.at` (which is actually intended for absolute times relative to the epoch) for duration formatting is problematic due to time zone issues. This wouldn't work at all for durations >24h, too.

Reply:
Come to think of it, doing human date formatting in bbb-events is out of scope, that should be done by whatever tool is consuming the bbb-events data and displaying it to the user.

Reply:
Removed.